Database/restaurant.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
+
+const restaurantSchema = new mongoose.Schema({
+  name: String,
+  ratings: [],
```

Comment:
Don't you need to specify what goes into the rating array? Seems inconsistent with the other schemas.